### DISCUSSION TITLE
Referencing wix 3 based wixlibs in wix 4/5 installers

### DISCUSSION THREAD

**Discussion by praaymak** [wixtoolset/issues#8650] [Questions] [answered]
Hello,

Just wondering if anyone has any experience with referencing wixlibs built using the wix 3 toolset in wix 4/ 5 installers? 

I took a quick look through the FAQ but didn't see anything that specifically talks about this scenario so feels like it should just work? When I attempt to do so I get an error that looks like this: 

Error	WIX0003	Attempted to load corrupt file from path: <path>. The file with format wixout contained unexpected content. Ensure the correct path was provided and that the file has not been incorrectly modified.

This could be due to various things but I want to know if, in general, this is supported. 

Cheers

**Comment by robmen** [comment, marked as answer]
> in general, this is supported.

Nope. Not even a little bit.

**Reply by praaymak** [reply]
@robmen thanks for the speedy reply.
